### ISSUE TITLE
Blog #2-2022 -April Blogs

### ISSUE THREAD

**Issue by bajpaigarima** [cdfoundation/community-ambassador-program#23]
We are starting our April blogs 
1. Re-architecting CD covering ideas from previous blog
2. Advancing CD with SREs
3  Securing CD with CDF Member projects  ( member project specific blog)

**Comment by enricforn** [comment]
@bajpaigarima  3rd bullet is referred to https://github.com/buildsec/ssf ?

**Comment by bajpaigarima** [comment]
@enricforn  yes, that can be one perspective  - we can try to write something

**Comment by enricforn** [comment]
https://docs.google.com/document/d/1I_YNt1bCHQ7ojtRPfF6M4vVJLeTtwQ5Iv96ig9rZjkQ/edit?usp=sharing

@bajpaigarima could you please take a look and send me feedback ?

**Comment by bradmccoydev** [comment]
@thschue and Andi might want to contribute or review blog 2
